modules/FaceExpressions_module/src/FaceExpressionsRetargeting.cpp

```diff
@@ -0,0 +1,181 @@
+/**
+ * @file FaceExpressionsRetargeting.cpp
+ * @authors Stefano Dafarra <stefano.dafarra@iit.it>
+ * @copyright 2021 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2021
+ */
+
+#include <FaceExpressionsRetargeting.hpp>
+#include <yarp/os/Vocab.h>
+#include <yarp/os/LogStream.h>
+#include <functional>
+#include <cmath>
+
+#define VAD_FREQUENCY     8000 //Other frequencies do not seem to work well
+#define VAD_SAMPLE_LENGTH 80 //This seeemed to be the only value for which there was a usable output.
+
+bool FaceExpressionsRetargetingModule::configure(yarp::os::ResourceFinder &rf)
+{
+    std::lock_guard<std::mutex> lock(m_mutex);
```

Comment:
Since, the process is single thread, is there a specific reason you added the mutex lock?